bin/binarize.sh

```diff
@@ -79,6 +83,11 @@ if [ -z "${user_name}" ]; then
    user_name=$(stat -c '%U' $workspace_path)
 fi
 
+# If the user workspace overlay path is not specified, use the default one
+if [ -z "${user_overlay_workspace}" ]; then
+   user_overlay_workspace="/home/user/workspace"
```

Comment:
Would suggest doing this for the sake of future proofing the code. I know we probably will never go away from using user but if we do, the code would break

Reply:
https://github.com/shadow-robot/sr-build-tools/pull/709/commits/c2cceb297e8f17b618ba368ec7336e727414ef3e

Reply:
How is `$(stat -c '%U' $workspace_path)` different from `$USER`?

Reply:
It finds the folder's owner `/home/user/projects/shadow_robot/base`, which should return "user".
At the point this script is being run I'm not sure $USER is being exported correctly without further testing